infrastructure/template.yaml

```diff
@@ -114,6 +128,12 @@ Resources:
       CodeSigningConfigArn:
         !If [EnforceCodeSigning, !Ref CodeSigningConfigArn, !Ref AWS::NoValue]
 
+  CiMappingFunctionLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub /aws/lambda/${CiMappingFunction}
+      RetentionInDays: 30
```

Comment:
Can we do this instead       `RetentionInDays: !If [IsDevLikeEnvironment, 7, 30]`